io-uring-test/src/utils.rs

```diff
@@ -6,9 +6,17 @@ macro_rules! require {
         $test:expr;
         $( $cond:expr ; )*
     ) => {
-        let mut cond = true;
+        require!($test; min_kernel_version = 0,0; $( $cond; )*)
```

Comment:
There's no need to modify the macros, I think it's fine to write like this

```rust
require!{
    test;
    check_kernel_version("6.6.0");
}
```

Reply:
Ah, I see. Done.